Assets/MixedRealityToolkit-SDK/Scripts/ControllerVisualizer.cs

```diff
@@ -0,0 +1,74 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.InputSystem.Utilities;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
+using UnityEngine;
+using UnityEngine.XR.WSA.Input;
+
+namespace Microsoft.MixedReality.Toolkit.SDK
+{
+    public class ControllerVisualizer : MonoBehaviour
+    {
+        GameObject leftController;
+        GameObject rightController;
```

Comment:
Maybe this class should be a single instance on any controller being visualized?

Reply:
This is a pure test only (as highlighted in the commit), just a playgound while I test access.

The method here is already wrong and I need it to infer from the InputSystem, not direct from the controller.
Ulitmately though, both direct and input system will be required to cover advanced cases.